src/EnergyPlus/HVACFan.cc

```diff
@@ -630,6 +635,13 @@ namespace HVACFan {
 		localFlowRatio.resize( 2, 0.0 );
 		localRunTimeFrac.resize( 2, 1.0 );
 
+		Real64 massFlowFromFlowRatios = ( m_maxAirMassFlowRate * ( flowRatio1 * runTimeFrac1 + flowRatio2 * runTimeFrac2 ) );
+		if ( abs ( m_inletAirMassFlowRate - massFlowFromFlowRatios ) > DataHVACGlobals::SmallMassFlow ) {
+			if( !localMismatchErrorShown ) ShowWarningError( "Flow rate mismatch in calcSimpleSystemFan (only first instance is reported)" );
```

Comment:
This is just temporary to see if any of the test files are sending arguments that don't match the inlet node mass flow rate